test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/CompositeModelBinderTest.cs

```diff
@@ -65,6 +65,7 @@ public class CompositeModelBinderTest
             var bindingContext = new ModelBindingContext
             {
                 FallbackToEmptyPrefix = true,
+                IsTopLevelObject = true,
```

Comment:
Discussed offline. `FallbackToEmptyPrefix` should never be copied into a child `ModelBindingContext` and therefore should never be `true` when `IsTopLevelObject` is `false`. No need to check `IsTopLevelObject` in `CompositeModelBinder`.